src/contracts/Protocol/ClaimController.sol

```diff
@@ -49,11 +53,16 @@ contract ClaimController is Ownable, IClaimController, OnlyApi {
      * @param questionId The id of the question
      * @param claimLimit The limit for the amount of people that can claim the question
      */
-    function initializeQuestion(uint256 questionId, uint256 claimLimit) public onlyApi {
+    function initializeQuestion(
+        uint256 questionId,
+        uint256 claimLimit,
+        uint256 threshold
+    ) public onlyApi {
         claimLimits[questionId] = claimLimit;
+        metricThresholds[questionId] = threshold;
```

Comment:
need to add documentation for this param

Reply:
Will add documentation on approval, just in case anything changes etc

Reply:
I meant param documentation in the code itself for example
```* @param claimLimit The amount of claims per question.```

Reply:
Ohhh yes my bad -- adding right away